features/intl-plural-rules.yml

```
@@ -0,0 +1,18 @@
+name: Intl.PluralRules
+description: "The `Intl.PluralRules` API creates a locale-aware formatter that tells you which plural-related language rules apply based on a given number."
```

Optional suggestion here. I'm not sure this is really a formatter; I think it's more of a low-level tool for making a formatter. Also, I thought it'd be nice to include the search term "pluralization" though it requires a little bit of contortion to the sentence. If you don't like any of this, feel free to ignore.

```suggestion
description: "The `Intl.PluralRules` API creates a locale-aware object that tells you which of the language's pluralization rules apply based on a given number."
```